app/views/layouts/application.html.erb

```diff
@@ -32,6 +32,8 @@
   <% end %>
 <% end %>
 
+<%= image_tag("hpi_main_building_keyvisual_1020x420.jpg", :alt => "HPI Background", :class => 'banner') %>
```

Comment:
This image is way to small, it should have at least 1920 px in width in order to fit for various screen resolutions.